Yank/tests/test_experiment.py

```diff
@@ -772,8 +772,9 @@ def test_validation_correct_experiments():
         {'system': 'sys', 'protocol': 'absolute-binding', 'restraint': {
             'type': 'FlatBottom', 'well_radius': '5.2*nanometers', 'restrained_receptor_atom': 1644}},
         {'system': 'sys', 'protocol': 'absolute-binding', 'restraint': {
-            'type': 'Boresch', 'restrained_atoms': [1335, 1339, 1397, 2609, 2607, 2606],
```

Comment:
How did this test pass before?

Reply:
The schema did not check for restraints arguments at parsing time. The error, in this case, would have been thrown only at runtime. Might be something that we need to add.